src/routes/RemediationDetails.js

```diff
@@ -170,6 +179,27 @@ const RemediationDetails = ({
 
   const { status, remediation } = selectedRemediation;
 
+  const fetchDisabledValue = useCallback(async () => {
+    return Promise.resolve(
```

Comment:
Is the `Promise.resolve` really needed here? `axios.get` should already be an async function and return a promise.

Reply:
the promise doesnt get fulfilled, so when trying to access the value, its empty. It runs too quickly

Reply:
"It runs too quickly".... why does it run too quickly?